sources.cfg

```diff
@@ -6,3 +6,7 @@ extensions +=
     mr.developer
 
 auto-checkout =
+    ftw.theming
+
+[branches]
+ftw.theming = plone51
```

Comment:
🚧 As `ftw.mobile` requires `ftw.theming`, `ftw.mobile` can only be Plone 5 compatible as soon as `ftw.theming` is compatible too. So we should wait for https://github.com/4teamwork/ftw.theming/pull/84 to be finished and released.

@mbaechtold is this a problem for you?

Reply:
`ftw.theming` 1.11.0 has support for Plone 5.1, so we no longer need to install it from the feature branch.